examples/example_write_mineos_input.py

```diff
@@ -1,182 +0,0 @@
-# This file is part of BurnMan - a thermoelastic and thermodynamic toolkit for the Earth and Planetary Sciences
```

Comment:
I've merged the example into one example for both types of output example_writeout_for_synthetic_seismo.py (should have renamed this first and then adapted, but didn't know how to merge the files smartly afterwards)